doc/source/release/1.15.0-notes.rst

```diff
@@ -24,56 +24,353 @@ This release requires Python 3.10+ and NumPy 1.23.5 or greater.
+- COO, CSR, CSC and LIL formats now support the ``axis`` argument for
+  ``count_nonzero``.
+- Sparse arrays and matrices may now raise errors when initialized with
+  incompatible data types, such as ``float16``.
+- ``min``, ``max``, ``argmin``, and ``argmax`` now support computation
+  over nonzero elements only via the new ``explicit`` argument.
 
 
 `scipy.spatial` improvements
 ============================
+- ``Rotation.concatenate`` now accepts a bare ``Rotation`` object, and will
+  return a copy of it.
 
 
 `scipy.special` improvements
 ============================
-
+- The factorial functions ``special.{factorial,factorial2,factorialk}`` now
+  offer an extension to the complex domain by passing the kwarg
+  ``extend='complex'``. This is opt-in because it changes the values for
+  negative inputs (which by default return 0), as well as for some integers
+  (in the case of ``factorial2`` and ``factorialk``; for more details,
+  check the respective docstrings).
+- `scipy.special.zeta` now defines the Riemann zeta function on the complex
+  plane.
+- `scipy.special.softplus` computes the softplus function
+- The spherical Bessel functions (`scipy.special.spherical_jn`,
+  `scipy.special.spherical_yn`, `scipy.special.spherical_in`, and
+  `scipy.special.spherical_kn`) now support negative arguments with real dtype.
+- `scipy.special.logsumexp` now preserves precision when one element of the
+  sum has magnitude much bigger than the rest.
+- The accuracy of several functions has been improved:
+
+  - `scipy.special.ncfdtr` and `scipy.special.nctdtr` have been improved
+    throughout the domain.
+  - `scipy.special.hyperu` is improved for the case of ``b=1``, small ``x``,
+    and small ``a``.
+  - `scipy.special.logit` is improved near the argument ``p=0.5``.
+  - `scipy.special.rel_entr` is improved when ``x/y`` overflows, underflows,
+    or is close to ``1``.
+
+- `scipy.special.gdtrib` may now be used in a CuPy ``ElementwiseKernel`` on
+  GPUs.
+- `scipy.special.ndtr` is now more efficient.
 
 `scipy.stats` improvements
 ==========================
+- A new probability distribution infrastructure has been added for the
+  implementation of univariate, continuous distributions:
+
+  - Speed, accuracy, and memory advantages of the new infastructure are
+    detailed in
+  - `scipy.stats.Normal` represents the normal distribution with the new
+    interface. In most cases, its methods are faster and more accurate than
+    those of `scipy.stats.norm`.
+  - Use `scipy.stats.make_distribution` to treat an existing continuous
+    distribution (e.g. `scipy.stats.norm`) with the new infrastructure.
+    This can improve the speed and accuracy of existing distributions,
+    especially for methods not overridden with custom formulas in the
+    implementation.
+
+- `scipy.stats.Mixture` has been added to represent mixture distributions.
+- The new `scipy.stats.lmoment` calculates sample l-moments and l-moment
+  ratios.
```

Comment:
a cool thing that might be worth mentioning, is that these sample estimators are *unbiased* (which isn't the case for e.g. `np.std` or `scipy.stats.moment` in general)